tasks/pip.yml

```diff
@@ -71,6 +83,26 @@
     virtualenv_site_packages: yes
   notify: reload supervisor
 
+- name: Install development version of openwisp-network-topology
+  when: openwisp2_network_topology_dev
+  pip:
+    name: https://github.com/openwisp/openwisp-network-topology/tarball/master
+    state: latest
+    virtualenv: "{{ virtualenv_path }}"
+    virtualenv_python: "{{ openwisp2_python }}"
+    virtualenv_site_packages: yes
+  notify: reload supervisor
+
+- name: Install development version of django-netjsongraph
+  when: openwisp2_django_netjsongraph_dev
```

Comment:
should be: `when: openwisp2_network_topology and openwisp2_django_netjsongraph_dev`